HorizonEditor/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# CMakeList.txt : CMake project for HznApplication, include source and define
+# project specific logic here.
+#
+cmake_minimum_required (VERSION 3.15)
+
+set(HZNEDITOR_SOURCE_FILES
+	"src/Main.cpp"
+  "src/EditorLayer.cpp")
+
+set(HZNEDITOR_HEADER_FILES
+ "include/EditorLayer.h" )
+
+# 'HznApplication' target which would generate an executable.
+add_executable(HznEditor ${HZNEDITOR_SOURCE_FILES} ${HZNEDITOR_HEADER_FILES})
```

Comment:
Executable for the Game/Scene?

Reply:
This is the standalone editor that I am writing to test code.. and develop things.